keps/sig-node/2411-cri-container-log-rotation/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: CRI Container Log Rotation
+kep-number: 2411
+authors:
+  - "@umohnani8"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
```

Comment:
SIG Node is already listed, no need to have it twice :)

Reply:
Fixed